Source/Header/DayHeaderView.swift

```diff
@@ -4,7 +4,7 @@ import DateToolsSwift
 
 public final class DayHeaderView: UIView, DaySelectorDelegate, DayViewStateUpdating, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
   public private(set) var daysInWeek = 7
-  public let calendar: Calendar
+  public var calendar: Calendar
```

Comment:
Should there be some reaction to this event? e.g. re-drawing all of the subviews, or propagating the calendar change further?